### DISCUSSION TITLE
Response error on get request !!

### DISCUSSION THREAD

**Discussion by Master-Y0da** [fastapi/fastapi#7361] [Questions] [answered]
Hi, I'm a little desperate with this problem...I have documented it with details here:

[link](https://stackoverflow.com/questions/63004799/fastapi-get-request-shows-validation-error)

 But can't find any solution...please help !!

**Comment by sachaventura** [comment, marked as answer]
The validation logic has no idea that you are passing in a list of `MFuente` object and not one single `MFuente` object.
Try setting your `reponse_model` to `List[FuenteSerializer.MFuente]` as explained in the docs [here](https://fastapi.tiangolo.com/tutorial/extra-models/#list-of-models).

**Comment by Master-Y0da** [comment]
@sachaventura thank you !! working now!!

**Comment by sachaventura** [comment]
No worries! Don't forget to close the issue :)

**Comment by tiangolo** [comment]
Thanks for the help here @sachaventura ! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
